plugins/scaffolder/src/components/fields/MyGroupsPicker/MyGroupsPicker.tsx

```diff
@@ -62,7 +62,7 @@ export const MyGroupsPicker = (props: MyGroupsPickerProps) => {
 
     const { items } = await catalogApi.getEntities({
       filter: {
-        type: 'Group',
+        kind: 'Group',
```

Comment:
This makes sense to me, and I think its how we had it initially, but I am not 100% sure why it was changed to type would you by chance know @Rugvip ? Otherwise it seems fine to me 👍